tests/unit-tests/UserMergeScenariosTests.swift

```diff
@@ -491,7 +495,8 @@ class UserMergeScenariosTests: XCTestCase, AuthProvider {
                    XCTFail("Expected events to be logged but found nil")
                }
         
-        IterableAPI.setEmail("testuser123@test.com", merge: false)
+        let identityResolution = IterableIdentityResolution(replayOnVisitorToKnown: true, mergeOnAnonymousToKnown: false)
+        IterableAPI.setEmail("testuser123@test.com", nil, identityResolution)
```

Comment:
Is named parameter still there? How can reader know what is passed as nil? 
Can we still do : `setEmail( email: "testuser123@test.com", authToken: nil, identityResolution:identityResolution)`

Reply:
Thats gonna break implementation since it is a public method.